### ISSUE TITLE
Add support for css vars

### ISSUE THREAD

**Issue by diegolealco** [diegolealco/foku#3]
**Is your feature request related to a problem? Please describe.**

I would like to keep element styles applied using css variables (within ui components) consistent with values applied by foku when elements are animated.

**Describe the solution you'd like**

```ts
foku({ target: '.target', height: 'var(--height)' })
```

**Describe alternatives you've considered**
n/a

**Additional context**
The most likely way of doing this is through `getComputedStyle` which may incur a performance penalty.

**Comment by different55** [comment]
If you assume that anime's the only one doing the changing (since the point of the feature is to separate domains), then it seems like you could cache things to keep things speedy. Without reaching in with JavaScript, there's relatively few ways that variables can be updated, I think. Media queries and stylesheets loading or being newly added are the only ones I can think of, and I believe media query changes can be listened to with callbacks.

**Comment by diegolealco** [comment]
you're right, though variables can be update programatically with javascript by adding inline styles (which I think you implied with "stylesheets loading or being newly added").

I don't think this should be much of an issue though, we only need to get values when animations or timelines begin playing - I don't think there is a use case for loading latest values half-way through an animation.
